modules/hosted-control-planes-support-matrix.adoc

```diff
@@ -1,29 +0,0 @@
-// Module included in the following assemblies:
-//
-// * hosted-control-planes/index.adoc
-
-
-:_mod-docs-content-type: CONCEPT
-[id="hosted-control-planes-upgrading-scenarios_{context}"]
-= Upgrading scenarios for {hcp}
-
-Consider the following information before you upgrade:
-
-* You use bare metal as a management cluster platform.
-
-* You use Agent or KubeVirt as a Hosted cluster platform.
```

Comment:
I do not see this information in [Requirements to upgrade hosted control planes](https://76523--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-updating#hosted-control-planes-upgrading-requirements_hcp-updating),  so removing this should get a QE ack.

Reply:
Yes. I restructured the hosted control planes book recently. Earlier hosted-control-planes-support-matrix.adoc was placed in the index.adoc. But that information i.e _now_ the `hcp-updating-requirements.adoc` module is moved (renamed) to its own relevant assembly/topic. See https://docs.openshift.com/container-platform/4.15/hosted_control_planes/hcp-updating.html#hosted-control-planes-upgrading-requirements_hcp-updating.

Reply:
This content has undergone the QE approval already. In my honest opinion, I  do not think this needs a QE approval.

Reply:
Also the `hcp-updating-requirements.adoc`'s content is updated and improved.

Reply:
yes that could be confusing. Thanks for the review.